src/main/kotlin/org/jetbrains/research/transformations/Anonymization.kt

```diff
@@ -0,0 +1,75 @@
+@file:Suppress("UNCHECKED_CAST")
```

Comment:
Why do you use it? Because you use `var anonymNamesMap: MutableMap<String, String>`, but after `String` is not `String` for value? Maybe should you change the type for values in the map like` MutableMap<String, MutableMap<String, String>>` and delete this line?  Because here you can set type explicitly

Reply:
not for this case.
I did it, because in the einverseApply at the line of nodeMap declaration I have a cast for Any? object into MutableMap<String, String>. (And I need it, because I get metadata as an Any? object, not a map :(  )